Cython/Utility/CppConvert.pyx

```diff
@@ -39,11 +39,16 @@ cdef inline object {{cname.replace("PyObject", py_type, 1)}}(const string& s):
 
 cdef extern from *:
     cdef cppclass vector "std::vector" [T]:
-        void push_back(T&)
+        ctypedef size_t size_type
+
+        void reserve(size_type) except +
+        void push_back(T&) except +
 
 @cname("{{cname}}")
-cdef vector[X] {{cname}}(object o) except *:
+cdef vector[X] {{cname}}(object o) except +*:
```

Comment:
Don't think this should be `except +`. The exception is caught by Cython immediately and converted into a Python exception

Reply:
True I missed that